travis_configure.sh

```diff
@@ -104,7 +131,14 @@ run_tests()
             print_error_msg "Invalid build configuration"
             return 1
         fi
-
+    elif [[ "$TRAVIS_OS_NAME" == "osx" ]]; then
+        autoreconf -fi
+        ## Pass the root of the openssl installation directory
```

Comment:
For example [here](https://github.com/sociomantic-tsunami/libdrizzle-redux/pull/162/commits/646659d8d179315752cb7f08343d49acb119199c#diff-6617b6efad46e9b5faa25ee9f65826a0R87) you are using a single `#` for a single-line comment.